egs/librispeech/ASR/RESULTS.md

```diff
@@ -77,6 +77,18 @@ for m in greedy_search modified_beam_search fast_beam_search; do
 done
 ```
 
+#### Smaller model
+
+A smaller model (~20M params) is also available with configuration based on [this comment](https://github.com/k2-fsa/icefall/pull/745#issuecomment-1405282740). The WERs are:
+
+| decoding method      | chunk size | test-clean | test-other | comment             | decoding mode        |
+|----------------------|------------|------------|------------|---------------------|----------------------|
+| greedy search        | 320ms      | 3.94       | 9.79       | --epoch 30 --avg 9  | simulated streaming  |
+| modified beam search | 320ms      | 3.88       | 9.53       | --epoch 30 --avg 9  | simulated streaming  |
+
+You can find a pretrained model, training logs, decoding logs, and decoding
+results at: <https://huggingface.co/desh2608/icefall-asr-librispeech-pruned-transducer-stateless7-streaming-small>
```

Comment:
Could you also provide two scripts like
- https://huggingface.co/Zengwei/icefall-asr-librispeech-pruned-transducer-stateless7-streaming-2022-12-29/blob/main/exp/export.sh
- https://huggingface.co/Zengwei/icefall-asr-librispeech-pruned-transducer-stateless7-streaming-2022-12-29/blob/main/exp/jit_trace_export.sh

and also `tokens.txt` in
https://huggingface.co/desh2608/icefall-asr-librispeech-pruned-transducer-stateless7-streaming-small/tree/main/data/lang_bpe_500

and also some `test_wavs` like
https://huggingface.co/Zengwei/icefall-asr-librispeech-pruned-transducer-stateless7-streaming-2022-12-29/tree/main/test_wavs

Reply:
and also 3 other files like
- https://huggingface.co/Zengwei/icefall-asr-librispeech-pruned-transducer-stateless7-streaming-2022-12-29/blob/main/exp/encoder_jit_trace.pt
- https://huggingface.co/Zengwei/icefall-asr-librispeech-pruned-transducer-stateless7-streaming-2022-12-29/blob/main/exp/decoder_jit_trace.pt
- https://huggingface.co/Zengwei/icefall-asr-librispeech-pruned-transducer-stateless7-streaming-2022-12-29/blob/main/exp/joiner_jit_trace.pt

Reply:
The purpose of adding the above files is to make it easier to test the model in other projects like `sherpa`.

Reply:
Done.

Reply:
Thanks.

I will  try to convert it to https://github.com/k2-fsa/sherpa-ncnn
and run it on embedded systems, e.g., Raspberry Pi.